aiohttp/helpers.py

```diff
@@ -852,6 +853,15 @@ def set_cookie(
         self._cookies[name] = value
         c = self._cookies[name]
 
+        if DEBUG:
+            cookie_length = (
```

Comment:
The cookie length is being modified later on, after this if-block. Should it be moved?

Reply:
Yes, it should be moved.